### ISSUE TITLE
Review all fonts

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/featured_match#2]
I have detected several wrong font sizes and weights, please review.

Few examples in the image are 
<img width="594" alt="Screenshot 2021-10-06 at 09 07 48" src="https://user-images.githubusercontent.com/37311649/136164491-74997304-b2db-45b1-9b4c-26c5304d6244.png">
![Screenshot 2021-10-06 at 09 07 38](https://user-images.githubusercontent.com/37311649/136164497-f5c20089-c178-4ed9-abab-b36b33bcebd2.png)
attached but it seems there are more.

**Comment by jonsnowpt** [comment]
This font seems different:

<img width="554" alt="Screenshot 2021-10-09 at 15 25 42" src="https://user-images.githubusercontent.com/37311649/136662052-2c5cee05-e1d7-4fcc-887b-2c246f35c85d.png">

**Comment by jonsnowpt** [comment]
Are this fonts with those values?

<img width="567" alt="Screenshot 2021-10-09 at 15 27 54" src="https://user-images.githubusercontent.com/37311649/136662138-90978694-efe3-4a60-8507-084bc48b7997.png">

![Screenshot 2021-10-09 at 15 28 33](https://user-images.githubusercontent.com/37311649/136662174-2e70e616-2d5c-4169-91b9-ca25c6466ab7.png)

**Comment by migbash** [comment]
Yes, all fonts are exactly the same as in the design, it's just as it look like for some reason, this was an issue addressed in previous widgets also.
